### ISSUE TITLE
feature request: custom path for config file

### ISSUE THREAD

**Issue by amir-arad** [briankendall/devreorder#41]
Hi
I'm using devreorder as part of my LAN setup. for managability purposes, I'd like to be able to set a custom parameter for devreorder for when to look for the configuration file by default. 
so instead of running `mygame.exe` I could run `mygame.exe --devreorder_path="C:\My Folder"`
or, using an environment variable perhaps:
```
set devreorder_path=C:\My Folder
mygame.exe
```

is this possible?

**Comment by briankendall** [comment]
Yes, it's possible, and I think it's a good idea, but it would require making a change to devreorder. In the meantime, is it possible for you to create a symlink at `C:\ProgramData\devreorder\devreorder.ini` that points to the ini file you want to use?
